### DISCUSSION TITLE
cannot import name 'CommandSession' from 'nonebot'

### DISCUSSION THREAD

**Discussion by songdf666** [nonebot/discussions#134] [问答]
我仿照教程里的获取天气的代码，但是遇到这种问题
<img width="1257" alt="image" src="https://user-images.githubusercontent.com/81544526/166148075-61fdc587-1ef1-441a-9646-a23eb4a50e8c.png">
有大佬能帮忙看看吗？
之所以这么用，是因为我想通过session.current_arg_images()获取发送的图片的url

**Comment by StarHeartHunt** [comment]
你大概是看到 nonebot 1 的文档去了，用 nonebot 2 的话看 https://v2.nonebot.dev
